### PR TITLE
[Runtime] Implement str.isalpha(), str.isdigit(), str.isalnum()

### PR DESCRIPTION
**Pull request by corona10**
Implement str.isalpha(), str.isdigit(), str.isalnum().

### CODE REVIEW

File: runtime/str.go [hunk 1]
```diff
@@ -918,6 +933,57 @@ func strInterpolate(f *Frame, format string, values *Tuple) (*Object, *BaseExcep
...
+	return GetBool(true).ToObject(), nil
+}
+
+func strIsDigit(f *Frame, args Args, kwargs KWArgs) (*Object, *BaseException) {
+	if raised := checkMethodArgs(f, "isdigit", args, StrType); raised != nil {
+		return nil, raised
+	}
+	s := toStrUnsafe(args[0]).Value()
+	if len(s) == 0 {
+		return GetBool(false).ToObject(), nil
```
Comment (aisk) [inline runtime/str.go]: I think it's more suitable for just using the global bool instance `True` / `False.`

Comment (corona10) [inline runtime/str.go]: @aisk Ah awesome!! Thank you for your comment!!

File: runtime/str.go [hunk 2]
```diff
@@ -136,6 +136,18 @@ func hashString(s string) int {
 	return h
 }
 
+func isAlNum(c byte) bool {
```
Comment (trotterdylan) [inline runtime/str.go]: Please group helpers at the bottom of the file.

File: runtime/str_test.go [hunk 3]
```diff
@@ -347,6 +347,18 @@ func TestStrMethods(t *testing.T) {
 		{"find", wrapArgs("bar", "a", 0, -1), NewInt(1).ToObject(), nil},
 		{"find", wrapArgs("foo", newTestTuple("barfoo", "oo").ToObject()), nil, mustCreateException(TypeErrorType, "'find/index' requires a 'str' object but received a 'tuple'")},
 		{"find", wrapArgs("foo", 123), nil, mustCreateException(TypeErrorType, "'find/index' requires a 'str' object but received a 'int'")},
+		{"isalnum", wrapArgs("123abc"), True.ToObject(), nil},
+		{"isalnum", wrapArgs(""), False.ToObject(), nil},
+		{"isalnum", wrapArgs("#$%"), False.ToObject(), nil},
```
Comment (trotterdylan) [inline runtime/str_test.go]: Also include a test case where there some some characters are in alnum but others aren't, e.g. "abc#123". Same idea for isalpha and isdigit.

**Comment by corona10** [comment]
@trotterdylan 
Thanks for reviews!!
I updated it

PTAL

**Pull request merged**